exporter/containerimage/writer.go

```diff
@@ -50,7 +50,7 @@ type ImageWriter struct {
 	opt WriterOpt
 }
 
-func (ic *ImageWriter) Commit(ctx context.Context, inp exporter.Source, oci bool, refCfg cacheconfig.RefConfig, buildInfo bool, buildInfoAttrs bool, sessionID string) (*ocispecs.Descriptor, error) {
+func (ic *ImageWriter) Commit(ctx context.Context, inp exporter.Source, refCfg cacheconfig.RefConfig, sessionID string, opts *ImageWriterOpts) (*ocispecs.Descriptor, error) {
```

Comment:
`refCfg` not needed as inside opts.

If `opts` is a pointer then the function should handle `nil` value. If not a pointer then not needed.